app/controllers/ctc/questions/dependents/child_can_be_claimed_by_other_controller.rb

```diff
@@ -0,0 +1,32 @@
+module Ctc
+  module Questions
+    module Dependents
+      class ChildCanBeClaimedByOtherController < BaseDependentController
+        include AuthenticatedCtcClientConcern
+        layout "yes_no_question"
+
+        def self.show?(dependent)
+          return false unless dependent
+          dependent.qualifying_child_relationship? &&
+            dependent.meets_qc_age_condition_2020? &&
+            dependent.meets_qc_misc_conditions? &&
+            dependent.meets_qc_residence_condition_2020?
+        end
+
+        def self.model_for_show_check(current_controller)
+          current_resource_from_params(current_controller.visitor_record, current_controller.params)
+        end
+
+        def method_name
+          'can_be_claimed_by_other'
```

Comment:
This seems like it might be another situation where the text of the page is opposite of the column name:

the text is "Are you the only person who can claim %{dependent_name}?", if you say "yes" then we either want `can_be_claimed_by_other` to be FALSE, or to change the name to `cant_be_claimed_by_other` (?) or `can_only_be_claimed_by_me` (!) or `can_only_be_claimed_by_client` or `can_only_be_claimed_by_filer`

This seems like a substantial issue, and it seems worth fixing the column name.